src/databricks/labs/ucx/source_code/known.json

```diff
@@ -1977,6 +1977,900 @@
+    "spacy.lang.te.stop_words": [],
+    "spacy.lang.th": [],
+    "spacy.lang.th.lex_attrs": [],
+    "spacy.lang.th.stop_words": [],
+    "spacy.lang.th.tokenizer_exceptions": [],
+    "spacy.lang.ti": [],
+    "spacy.lang.ti.examples": [],
+    "spacy.lang.ti.lex_attrs": [],
+    "spacy.lang.ti.punctuation": [],
+    "spacy.lang.ti.stop_words": [],
+    "spacy.lang.ti.tokenizer_exceptions": [],
+    "spacy.lang.tl": [],
+    "spacy.lang.tl.lex_attrs": [],
+    "spacy.lang.tl.stop_words": [],
+    "spacy.lang.tl.tokenizer_exceptions": [],
+    "spacy.lang.tn": [],
+    "spacy.lang.tn.examples": [],
+    "spacy.lang.tn.lex_attrs": [],
+    "spacy.lang.tn.punctuation": [],
+    "spacy.lang.tn.stop_words": [],
+    "spacy.lang.tokenizer_exceptions": [],
+    "spacy.lang.tr": [],
+    "spacy.lang.tr.examples": [],
+    "spacy.lang.tr.lex_attrs": [],
+    "spacy.lang.tr.stop_words": [],
+    "spacy.lang.tr.syntax_iterators": [],
+    "spacy.lang.tr.tokenizer_exceptions": [],
+    "spacy.lang.tt": [],
+    "spacy.lang.tt.examples": [],
+    "spacy.lang.tt.lex_attrs": [],
+    "spacy.lang.tt.punctuation": [],
+    "spacy.lang.tt.stop_words": [],
+    "spacy.lang.tt.tokenizer_exceptions": [],
+    "spacy.lang.uk": [],
+    "spacy.lang.uk.examples": [],
+    "spacy.lang.uk.lemmatizer": [],
+    "spacy.lang.uk.lex_attrs": [],
+    "spacy.lang.uk.stop_words": [],
+    "spacy.lang.uk.tokenizer_exceptions": [],
+    "spacy.lang.ur": [],
+    "spacy.lang.ur.examples": [],
+    "spacy.lang.ur.lex_attrs": [],
+    "spacy.lang.ur.punctuation": [],
+    "spacy.lang.ur.stop_words": [],
+    "spacy.lang.vi": [],
+    "spacy.lang.vi.examples": [],
+    "spacy.lang.vi.lex_attrs": [],
+    "spacy.lang.vi.stop_words": [],
+    "spacy.lang.xx": [],
+    "spacy.lang.xx.examples": [],
+    "spacy.lang.yo": [],
+    "spacy.lang.yo.examples": [],
+    "spacy.lang.yo.lex_attrs": [],
+    "spacy.lang.yo.stop_words": [],
+    "spacy.lang.zh": [],
+    "spacy.lang.zh.examples": [],
+    "spacy.lang.zh.lex_attrs": [],
+    "spacy.lang.zh.stop_words": [],
+    "spacy.language": [],
+    "spacy.lookups": [],
+    "spacy.matcher": [],
+    "spacy.ml": [],
+    "spacy.ml._character_embed": [],
+    "spacy.ml._precomputable_affine": [],
+    "spacy.ml.callbacks": [],
+    "spacy.ml.extract_ngrams": [],
+    "spacy.ml.extract_spans": [],
+    "spacy.ml.featureextractor": [],
+    "spacy.ml.models": [],
+    "spacy.ml.models.entity_linker": [],
+    "spacy.ml.models.multi_task": [],
+    "spacy.ml.models.parser": [],
+    "spacy.ml.models.span_finder": [],
+    "spacy.ml.models.spancat": [],
+    "spacy.ml.models.tagger": [],
+    "spacy.ml.models.textcat": [],
+    "spacy.ml.models.tok2vec": [],
+    "spacy.ml.staticvectors": [],
+    "spacy.ml.tb_framework": [],
+    "spacy.pipe_analysis": [
+      {
+        "code": "table-migrate",
+        "message": "Can't migrate 'msg.table(body, header=header, divider=True, multiline=True)' because its table name argument cannot be computed"
```

Comment:
this is not `spark.table`

Reply:
removed, filed https://github.com/databrickslabs/ucx/issues/2017